crates/agentgateway/src/http/jwt.rs

```diff
@@ -85,12 +109,28 @@ impl Debug for Jwt {
 	}
 }
 
+#[derive(Debug, Clone, serde::Deserialize)]
+#[serde(untagged, rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "schema", derive(JsonSchema))]
+pub enum LocalJwtConfig {
+	Multi {
+		#[serde(default)]
+		mode: Mode,
+		providers: Vec<ProviderConfig>,
+	},
+	Single {
+		#[serde(default)]
+		mode: Mode,
+		issuer: String,
+		audiences: Vec<String>,
+		jwks: serdes::FileInlineOrRemote,
```

Comment:
very nit: I think you could do 
```rust
#[serde(flatten)
provider: ProviderConfig,
```

though I have had issues with flatten+enum so maybe not